### ISSUE TITLE
Fix Request: false positive in AvoidUnusedAsignments

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#221]
Example:
```
        licenseResponses = new LicenseResponse[1];
        licenseResponse = new LicenseResponse();
        licenseResponses[0] = licenseResponse;
```
False positive for 2 reasons: 
1. array and array element are mixed up
2. the are fields, not local vars
